src/app/core/social-container/social-container.component.html

```diff
@@ -7,7 +7,8 @@
       <li><a href="https://www.linkedin.com/company/ripio-credit-network/" matTooltip="Linkedin" matTooltipPosition="above" target="_blank"><i class="fab fa-linkedin-in fa-lg"></i></a></li>
       <li><a href="https://www.reddit.com/r/rcn_token/" matTooltip="Reddit" matTooltipPosition="above" target="_blank"><i class="fab fa-reddit fa-lg"></i></a></li>
       <li><a href="https://github.com/ripio/rcn.loans" matTooltip="Github" matTooltipPosition="above" target="_blank"><i class="fab fa-github fa-lg"></i></a></li>
-      <li><a href="{{ linkContract }}" matTooltip="Github" matTooltipPosition="above"><i class="fas fa-link fa-lg"></i></a></li>
+      <li><a href="https://discordapp.com/channels/524313273512755211/524313273512755217" matTooltip="Discord" matTooltipPosition="above" target="_blank"><i class="fab fa-discord fa-lg"></i></a></li>
```

Comment:
I am not sure if `https://discordapp.com/channels/524313273512755211/524313273512755217` is a valid invite link, maybe we can use `https://discord.gg/62UCxbS`